netmap/attributes.go

```diff
@@ -126,11 +126,24 @@ var errNoSubnets = errors.New("no subnets")
 func IterateSubnets(node *NodeInfo, f func(refs.SubnetID) error) error {
 	attrs := node.GetAttributes()
 
+	type zeroStatus uint8
+
+	const (
+		_ zeroStatus = iota
+		// missing attribute of zero subnet
+		zeroNoAttr
+		// with `False` attribute
+		zeroExit
+		// with `True` attribute
+		zeroEntry
+	)
+
```

Comment:
Can we use `zeroVal` instead? It doesn't require new type and `NoAttr` state is `zeroVal == ""`.

Reply:
I see it is removed later.